app/controllers/candidate_interface/unsubmitted_application_form_controller.rb

```diff
@@ -61,10 +61,14 @@ def redirect_to_application_if_between_cycles
       true
     end
 
-    def redirect_to_carry_over_if_unsubmitted_previous_cycle
-      return unless current_application.must_be_carried_over?
+    def redirect_to_carry_over
+      return unless must_be_carried_over?
 
       redirect_to candidate_interface_start_carry_over_path
     end
+
+    def must_be_carried_over?
```

Comment:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/5159/files#r670329247